src/mlpack/core/dists/discrete_distribution.hpp

```diff
@@ -41,7 +41,6 @@ namespace distribution /** Probability distributions. */ {
  * (size_t) as observations, these can be converted to doubles (which is what
  * arma::vec holds).  This distribution internally converts those doubles back
  * into size_t before comparisons.
- * @endnote
```

Comment:
Huh, didn't know that `@endnote` wasn't a valid command.

Reply:
It would make more sense if it were, but it apparently just uses end of comment of next empty line. :shrug: